src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,65 @@
+package duke;
+
+public class Parser {
+    public void getCommand (String input,Storage store, Ui ui) throws DukeException{
+        ui.printLine();
+        String command, others;
+        if(input.contains(" ")){
+            command = input.split(" ",2)[0];
+            others = input.split(" ",2)[1];
+        }else{
+            command = input;
+            others = "";
+        }
+        switch (command) {
```

Comment:
I like this part. Using "switch" to compare the input can maximise readability and easier for user to read